modules/common/react/lib/utils/convertPseudoStatesBehavior.ts

```diff
@@ -0,0 +1,62 @@
+import {CSSObject, Interpolation} from '@emotion/core';
+import {WithTheme} from '@emotion/styled';
+import {Overwrapped} from '@emotion/styled-base/types/helper';
+
+/**
+ * Allows the ability to force a state by a pseudo selector by changing the selector into a class creating a new styled object with the class names
+ * @param shouldChangePseudoSelectors If true, it will replace pseudo selectors from the styled object and change them to classes
+ * @param obj The styled object passed to the function to replace any pseudo selectors with a class name
+ */
+
+export interface ConvertPseudoStatesBehaviorProps {
```

Comment:
Sorry, I meant without the `Props` at the end. I think we generally only add `Props` when it's an interface for a components whole set of props (i.e. `Themeable` and `GrowthBehavior` don't have this)

Reply:
ahh i si, i think i just got use to naming them props, will rename!